.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 .DS_Store
 Thumbs.db
+/localCopyMatchmaker
```

Comment:
Remove this. I know I added it before, but we don't actually need it.

Reply:
![gitignore](https://user-images.githubusercontent.com/36285415/54785627-14d7b200-4bfd-11e9-8874-9b0e8132e669.JPG)

Reply:
i don't see anything